qiita_pet/templates/analysis_results.html

```diff
@@ -12,6 +12,19 @@
 
 <div class="row">
   <h1>&nbsp;Analysis: <u>{{aname}}</u></h1>
+  {% if status == "altered_data" %}
```

Comment:
Probably useful to use an accordion-like visualization here. The problem is that if a lot of thing change, it will be really hard to go over the list.